stratum/hal/bin/barefoot/deb/postinst

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
 # Copyright 2020-present Open Networking Foundation
 # SPDX-License-Identifier: Apache-2.0
-# Set up the huge page
-echo "vm.nr_hugepages = 128" >> /etc/sysctl.conf
-sysctl -p /etc/sysctl.conf
-mkdir -p /mnt/huge
-mount -t hugetlbfs nodev /mnt/huge
+
+# If not running in Docker,
+if ! cat /proc/1/cgroup | grep -q "/docker"; then
+  # Set up huge pages
+  echo "vm.nr_hugepages = 128" >> /etc/sysctl.conf
+  sysctl -p /etc/sysctl.conf
+  mkdir -p /mnt/huge
+  mount -t hugetlbfs nodev /mnt/huge
+fi
 
 # Reload libraries
 ldconfig
 
-# Enables Stratum service
-systemctl enable stratum_bf.service
+# If systemd is installed,
+if command -v systemctl &> /dev/null
+then
+  # Enable Stratum service
+  systemctl enable stratum_bf.service
+fi
```

Comment:
Did you know [`dh_systemd_enable`](https://manpages.debian.org/testing/debhelper/dh_systemd_enable.1.en.html)?

Never used it, but might be applicable to our use case.

Reply:
The intention of this block is to skip installing the service file if the user doesn't have systemd (e.g. in the Docker container). I haven't heard of `dh_systemd_enable` but we could see if it does what we need.

Reply:
I think dh_systemd_enable is just to help in the debian packaging process, which is managed by Bazel. It would then autogenerate code for the postinst, prerm, etc. scripts similar to what we have.

Reply:
It seems like there is a recommendation to use `deb-systemd-helper` in maintscripts, though
https://people.debian.org/~stapelberg/debconf13-making-your-package-work-with-systemd.pdf

Reply:
Looks like a good idea, but should be applied to all platforms then. Out of scope for this PR.